collector/pg_replication.go

```diff
@@ -64,7 +63,8 @@ var (
 	END as is_replica`
 )
 
-func (c *PGReplicationCollector) Update(ctx context.Context, db *sql.DB, ch chan<- prometheus.Metric) error {
```

Comment:
weird, I thought that we fixed this in another PR.

Reply:
I think this was a separate collector that got merged midway.